components/search/SearchSuggestion.vue

```diff
@@ -26,7 +26,20 @@
                 <div
                   class="is-flex is-flex-direction-row is-justify-content-space-between pt-2 pr-2">
                   <span class="main-title name">{{ item.name }}</span>
-                  <span>{{ urlPrefix.toUpperCase() }}</span>
+                  <span class="has-text-grey">
+                    {{ urlPrefix.toUpperCase() }}
+                  </span>
+                </div>
+                <div
+                  class="is-flex is-flex-direction-row is-justify-content-space-between pr-2">
```

Comment:
```suggestion
                  class="is-flex is-justify-content-space-between pr-2">
```

flex-direction: row is default